CrowdStrikeFalcon/crowdstrike_falcon/custom_iocs.py

```diff
@@ -110,7 +110,7 @@ def run(self, arguments):
         stix_objects = self.json_argument("stix_objects", arguments)
         indicators = self.get_valid_indicators(stix_objects)
         if len(indicators["valid"]) == 0 and len(indicators["revoked"]) == 0:
-            self.log("Received indicators were not valid and/or not supported")
+            self.log(f"Received indicators were not valid and/or not supported in stix_objects {str(stix_objects)}")
```

Comment:
I'm afraid that `stix_object` will be huge for the logger.
Maybe the following code will be more suitable:
```suggestion
            self.log("Received indicators were not valid and/or not supported in stix_objects", level="error", stix_object=stix_objects)
```

Reply:
@Darkheir What do you think about the original proposal and the suggestion? Thanks

Reply:
Ok, I commit your suggestion